packages/api/test/e2e/api.create.e2e.ts

```diff
@@ -21,7 +21,8 @@ describe('e2e api create', () => {
   let api;
   let incorrectApi;
 
-  it('For Azalea chain - checking if static metadata is same as latest', async () => {
+  // TODO - enable this test on runtime upgrade on Azalea
+  it.skip('For Azalea chain - checking if static metadata is same as latest', async () => {
```

Comment:
should reconsider the usefulness of this test as it's kind of a chicken-egg thing.
I think a dev release image would have same metadata.
Ideally API release can go out same time as the upgrade

Reply:
It could be better to be a CircleCI script test for the release.

Reply:
updated the test case for dev node.. will create a ticket to have a test in circle ci..